### PR TITLE
Add additional profile counters from NWX

### PR DESCRIPTION
**Pull request by yumnuska**

### CODE REVIEW

File: SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp [hunk 1]
```diff
@@ -1152,9 +1158,13 @@ void USpatialActorChannel::RemoveRepNotifiesWithUnresolvedObjs(TArray<UProperty*
 
 void USpatialActorChannel::ServerProcessOwnershipChange()
 {
-	if (!IsAuthoritativeServer())
+	SCOPE_CYCLE_COUNTER(STAT_ServerProcessOwnershipChange);
 	{
-		return;
+		SCOPE_CYCLE_COUNTER(STAT_IsAuthoritativeServer);
```
Comment (m-samiec) [inline SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp]: This counter will be the same as above, what's the need?

Comment (yumnuska) [reply SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp]: It's in a smaller scope, so it catches just the IsAuthoritativeServer. The parent scope includes the ownership changes

Comment (yumnuska) [reply SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp]: although I agree that the time for the inner scope should be negligible.

Comment (m-samiec) [reply SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp]: Oh, I read this as a counter, not a cycle_counter.

File: SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp [hunk 2]
```diff
@@ -1172,7 +1182,11 @@ void USpatialActorChannel::ServerProcessOwnershipChange()
 
 void USpatialActorChannel::UpdateEntityACLToNewOwner()
 {
-	FString NewOwnerWorkerAttribute = SpatialGDK::GetOwnerWorkerAttribute(Actor);
+	FString NewOwnerWorkerAttribute;
+	{
+		SCOPE_CYCLE_COUNTER(STAT_GetOwnerWorkerAttribute);
```
Comment (m-samiec) [inline SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp]: Move this inside `SpatialGDK::GetOwnerWorkerAttribute` ?

Comment (m-samiec) [reply SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp]: Total nit, feel free to dismiss.

Comment (yumnuska) [reply SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp]: makes sense, will do. My guess is it was done this way to minimize divergence on our end.

Comment (yumnuska) [reply SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp]: On second thought... that would mean either moving the impl for GetOwnerWorkerAttribute out of the header, or adding an include for SpatialNetDriver in order to pick up the decl for the stat group. It's probably worth a discussion whether we want to toss all of these stats into that group at all, or should define new groups. Will leave this PR until we can chat in the new year.

Comment (yumnuska) [reply SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp]: Merged in as-is; we can chat about the bigger picture and adjust later.

**Pull request merged**